cmd/panacead/testnet.go

```diff
@@ -186,81 +188,83 @@ func initTestnet(config *tmconfig.Config, cdc *codec.Codec) error {
 		}
 
 		// save private key seed words
-		err = writeFile(fmt.Sprintf("%v.json", "key_seed"), clientDir, cliPrint)
-		if err != nil {
+		if err := writeFile(fmt.Sprintf("%v.json", "key_seed"), clientDir, cliPrint); err != nil {
 			return err
 		}
 
-		coin := sdk.TokensFromTendermintPower(500) // TODO @ggomma check power - coin relation
-		accs = append(accs, app.GenesisAccount{
+		accTokens := sdk.TokensFromConsensusPower(1000)
+		accStakingTokens := sdk.TokensFromConsensusPower(500)
+		accs = append(accs, genaccounts.GenesisAccount{
 			Address: addr,
 			Coins: sdk.Coins{
-				sdk.NewCoin(assets.MicroMedDenom, coin),
+				sdk.NewCoin(fmt.Sprintf("%stoken", nodeDirName), accTokens),
```

Comment:
remove this line? what's the difference between accTokens and accStakingTokens?

Reply:
We don't need that. With that, the `genesis.json` for testnet is created like below, but I don't think we need the `node*token`.
```
    "accounts": [
      {
        "address": "panacea1gwrva7xvr38gugcfgtx4yz5uwddvwtymvp9rcg",
        "coins": [
          {
            "denom": "node0token",
            "amount": "1000000000"
          },
          {
            "denom": "umed",
            "amount": "500000000"
          }
        ],
```